frontend/TestRun/SCT/SctEvent.svelte

```diff
@@ -0,0 +1,194 @@
+<script lang="ts">
+    import humanizeDuration from "humanize-duration";
+    import { timestampToISODate } from "../../Common/DateUtils";
+    import type { EventSeverityFilter, Options, SCTEvent } from "./SctEvents.svelte";
+    import { faNode } from "@fortawesome/free-brands-svg-icons";
+    import Fa from "svelte-fa";
+    import { faClipboard, faClock, faServer, faSpider } from "@fortawesome/free-solid-svg-icons";
+
+    interface Props {
+        event: SCTEvent,
+        filterState: EventSeverityFilter,
+        options: Options,
+    }
+    let { event, filterState, options }: Props = $props();
+
+    const sliceMessage = function (message: string): string {
+        return fullMessage ? message : message.slice(0, Math.max(Math.ceil(message.length * 0.15), 500));
```

Comment:
500 is magic number - please add it to const with description.
Btw. this approach is bad when we have message with length of 501 - we add ... to uncover one character.
I'd say we should have margin, e.g. if message is greater than 600 characters, slice by 500